.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
Mind also removing this in the `package.json` - https://github.com/solid/solid-client/blob/master/package.json#L27-L30